README.md

```diff
@@ -257,6 +257,8 @@ isOpen|N/A|boolean|Returns `true` if circuit is open
 - **healthCheckInterval**: time in `ms` interval between each execution of health check function
 - **healthCheck**: function to call for the health check (can be defined also with calling `healthCheck` function)
 - **fallback**: function to call for fallback (can be defined also with calling `fallback` function)
+- **isPromise**: `boolean` to opt out of check for callback in function
```

Comment:
We should add something in here that tells the user that if they pass this the flag will also be applied to the`healthCheck` and `fallback`. I don't think that will be obvious to an end-user.